homeassistant/components/moon/config_flow.py

```diff
@@ -23,13 +22,6 @@ async def async_step_user(
             return self.async_abort(reason="single_instance_allowed")
 
         if user_input is not None:
-            return self.async_create_entry(
-                title=user_input.get(CONF_NAME, DEFAULT_NAME),
-                data={},
-            )
+            return self.async_create_entry(title="", data={})
```

Comment:
Why do we change the default title?

Reply:
Moon is translated, it was originally kept to accommodate the custom at title on import.

This remove the translated integration name with an untranslated version (title) shown underneath it, showing just the translated integration name.

Reply:
Ok, isn't this a separate change though? It's not really related to removing the import.

Reply:
Sorry, I think it was, as we don't have to keep that compatibility anymore for the imports. I'm happy to take it out in a separate PR. Let me know.

Reply:
Yeah, I'm not in favor of the use of empty string for this purpose. I suggest we break it out so we can merge the rest.